paddle/fluid/framework/ir/mkldnn/cpu_bfloat16_placement_pass.cc

```diff
@@ -56,7 +56,11 @@ int CPUBfloat16PlacementPass::SetMkldnnDataType(ir::Graph* graph) const {
     // Only float input can be converted to bfloat16
     if (op_in->Var()->GetDataType() != proto::VarType::FP32) return;
 
-    if ((op->Op()->HasAttr("mkldnn_data_type") ||
+    bool op_always_included =
```

Comment:
It is possible that there are more BF16 operators which, due to the fact that they have `mkldnn_data_type` added dynamically, are not used. But I will create a separate task to do this.

Reply:
Although I think you can remove this check for `mkldnn_data_type` at all and just leave `if (!platform::HasOpINT8DataType(op->Op()))`  This check was needed earlier because it would be a mistake in assigning value for attribute that doesn't exits and now it is no longer needed.